start.sh

```diff
@@ -0,0 +1 @@
+mdbook serve
```

Comment:
This is to reminder new contributors about the required command to deploy the website locally. This is an initiative, we could add more commands or change this one as the we go. 💡

Reply:
if theres only one command i'd disagree to make a start.sh for it